packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelMain.tsx

```diff
@@ -127,7 +127,7 @@ function NetworkListbox({
         {!disabled && <ChevronDownIcon className="h-4 w-4" />}
       </Listbox.Button>
 
-      <Listbox.Options className="absolute z-20 mt-2 rounded-xl bg-white shadow-[0px_4px_12px_#9e9e9e]">
+      <Listbox.Options className="absolute z-20 mt-2 overflow-hidden rounded-xl bg-white shadow-[0px_4px_12px_#9e9e9e]">
```

Comment:
@bartlomiej94 i had to add overflow hidden because it currently overflows
<img width="246" alt="image" src="https://user-images.githubusercontent.com/13184582/233609415-21097539-644e-43cb-a50b-09641ded4108.png">